presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -134,6 +134,16 @@ class TaskManager {
     return taskMap_.rlock()->size();
   }
 
+  /// Stores the processed input data size in bytes for tasks.
```

Comment:
Nit : Move this code to cpp file.

Also, documentation should be "Returns the processed "